lib/tasks/data/check_missing_required_ids_in_tax_rates.rake

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+namespace :ofn do
+  namespace :data do
+    desc 'Checking missing required ids in Spree::TaxRate'
+    task check_missing_required_missing_ids_in_spree_tax_rates: :environment do
+      puts 'Checking for null tax_category_id'
+      ids = Spree::TaxRate.where(tax_category_id: nil).pluck(:id)
+
+      if ids.empty?
+        puts 'No NULL tax_category_id found in spree_tax_rates'
+      else
+        puts 'NULL tax_category_ids s have been found in spree_tax_rates:'
+        print ids
+      end
+    end
```

Comment:
Thank your for providing this. But having it in this commit doesn't really help. The rake task can only be run once the code has been deployed. But if you deploy this commit then the migration is run already.

I did a pre-flight check querying all servers via Ansible. Only admins can do that, of course. I'm not sure if there's any better approach.

Can you remove this file again? Or maybe I quickly do it to save time.

Reply:
Hello @mkllnk , I will make these scripts in a separate commits ahead of migrations for next time.
Also, I will make a PR to delete these one-short rake taskes I have created.